features/modified_execution_paths/class_method_call_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../feature_helper'
+
+describe 'Changing source file' do
+  subject(:forecast) do
+    Crystalball.foresee(workdir: root, map_path: root.join('execution_map.yml')) do |predictor|
+      predictor.use Crystalball::Predictor::ModifiedExecutionPaths.new
+    end
+  end
+  include_context 'simple git repository'
+
+  it 'adds class2 example when class1 changes because of class method call' do
+    change class1_path
+
+    is_expected.to include(
+      './spec/class2_spec.rb[1:4:1]'
+    )
```

Comment:
Please reuse [this](https://github.com/toptal/crystalball/pull/68/files#diff-0fe6e382aea1d1671075e5319b722f6cR7) when it will be merged